### PR TITLE
 webView: Implement longPress method.

### PR DESCRIPTION
**Pull request by jainkuniya**
final part of #1680 & long press 😄

### CODE REVIEW

**Comment by jainkuniya** [comment]
Updated :+1:

**Comment by jainkuniya** [comment]
Fixed flow errors 👍

File: src/render-html/webViewEventHandlers.js [hunk 1]
```diff
@@ -92,6 +112,44 @@ export const handleImage = (props: Props, event: MessageListEventImage) => {
   }
 };
 
+export const handleLongPress = (
```
Comment (borisyankov) [inline src/render-html/webViewEventHandlers.js]: Why are you not redirecting this to `MessageListContainer`. You don't need to recreate it!

Comment (jainkuniya) [reply src/render-html/webViewEventHandlers.js]: Moved to `MessageListContainer`. Now both `render-html` & `render-native` shares same code.

**Comment by borisyankov** [comment]
Looks good!

**Pull request merged**